apps/worker/src/app/workflow/usecases/send-message/send-message-push.usecase.ts

```diff
@@ -273,59 +283,24 @@ export class SendMessagePush extends SendMessageBase {
   }
 
   private async sendMessage(
+    command: SendMessageCommand,
+    message: MessageEntity,
     subscriber: IPushOptions['subscriber'],
     integration: IntegrationEntity,
-    target: string[],
+    deviceToken: string,
```

Comment:
Now we will miss the functionalities of sending multiple tokens at the same time in exchange of better execution details without doing breaking changes in the providers.
We need to implement auxiliary methods in the providers to handle the multiple token and to handle properly their responses.